lib/utils/platform_service.dart

```diff
@@ -0,0 +1,65 @@
+import "dart:io";
+import "package:flutter/services.dart";
+import "package:simple_logger/simple_logger.dart";
+
+/// Platform Service
+///
+/// This class provides a set of platform specific services.
+///
+/// At the moment only Android is supported.
+class PlatformService {
+  static const _methodChannelName =
+      "fi.metatavu.noheva_visitor_ui/platform_service";
+  static const _setWMDensityMethod = "setWMDensity";
+  static const _getWMDensityMethod = "getWMDensity";
+  static const _restartActivityMethod = "restartActivity";
+  static const _setWMDensityArgument = "density";
+  static const _channel = MethodChannel(_methodChannelName);
+
+  /// Sets VM density to [density]
+  static Future<void> setWMDensity(double density) async {
```

Comment:
Why do we set density as double but return it as an int?

Reply:
API returns it as double but Android accepts it as an integer.

Resolved by parsing it in the DeviceSettings::_applyScreenDensitySetting